services/web-app/utils/url.js

```diff
@@ -29,3 +29,13 @@ export const urlsMatch = (url1, url2, matchChunks = 0) => {
     JSON.stringify(urlObj2.pathname.split('/', matchChunks))
   )
 }
+
+/**
+ * Determines if a url string is absolute
+ *
+ * @param {string} url url string to evaluate
+ * @returns {boolean} whether the url is absolute or not
+ */
+export const isAbsoluteUrl = (url) => {
+  return url.startsWith('http://') || url.startsWith('https://')
+}
```

Comment:
Though somewhat rare, there are other protocols that could be used besides http and https. It would be more comprehensive to try to construct a URL from the given input (`new URL(...)`) and catch an exception indicating an invalid absolute url. Something like:

```ts
let absoluteUrl

try {
  absoluteUrl = new URL(url)
} catch {}

return !!absoluteUrl
```

This might mean that you'd want to move this logic back up to the image.js file (assuming this utility function isn't/won't be used anywhere else).

Reply:
found a similar function in `@utils/string` and using that instead now